PullRequestCreated/ReadMe.md

```diff
@@ -46,7 +50,7 @@ jobs:
     - uses: sonarsource/gh-action-lt-backlog/PullRequestCreated@v2
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
-          jira-project: NET
           jira-user: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_USER }}
           jira-token: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}
+          jira-project: NET
```

Comment:
Let's don't get accidental issues in NET from people brainlessly copy-pasting templates :D
```suggestion
          jira-project: EXAMPLE
```